test/py/requirements.txt

```diff
@@ -0,0 +1,24 @@
+#####################################################################################
+# The MIT License (MIT)
+#
+# Copyright (c) 2015-2023 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#####################################################################################
+
```

Comment:
Why is this empty? Dont we need numpy installed?

Reply:
The output for pip list:
```
test/py/venv/3.8/bin/pip list
Package       Version
------------- -------
pip           20.0.2 
pkg-resources 0.0.0  
setuptools    44.0.0
```
I also uninstalled `numpy` on the docker container I was testing things on, even though tests run on a virtual environment.
I will also setup a docker container by removing the packages that are installed in `install_prereqs` and do a run with that.

Reply:
if the tests are passing, then I guess you don't need numpy

Reply:
We do need numpy for the tests. They dont fail because we silently skip the test if numpy is not found: https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/test/py/test_gpu.py#L27

We should probably remove this try/except clause and then make sure we have the numpy dependencies.

Reply:
> We do need numpy for the tests. They dont fail because we silently skip the test if numpy is not found
Thanks for catching this Paul, I had not realized that this was the case. I have added numpy in the requirements file.

> We should probably remove this try/except clause and then make sure we have the numpy dependencies.
I can remove the try/except clause in a separate PR. I am open to doing it in this one if you think we should make the change here.

Reply:
> I can remove the try/except clause in a separate PR. I am open to doing it in this one if you think we should make the change here.

Here is the PR for removing the try/except clause: https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/2316
@pfultz2

Reply:
So that has been merged, so can the requirements.txt be updated to list numpy?

Reply:
So that has been merged, so can the requirements.txt be updated to list numpy?

> Yes, that is done.